website/docs/r/org_user.html.markdown

```diff
@@ -0,0 +1,165 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_org_user"
+sidebar_current: "docs-vcd-resource-org-user"
+description: |-
+  Provides a vCloud Director Organization user. This can be used to create, update, and delete organization users.
+---
+
+# vcd\_org\_user
+
+Provides a vCloud Director Org User. This can be used to create, update, and delete organization users, including org administrators.
+
+Supported in provider *v2.4+*
+
+~> **Note:** Only `System Administrator` or `Org Administrator` users can create users.
+
+## Example Usage
+
+```hcl
+resource "vcd_org_user" "my-org-admin" {
+  org = "my-org"
+
+  name          = "my-org-admin"
+  description   = "a new org admin"
+  role          = "Organization Administrator"
+  provider_type = "INTEGRATED"
+  password      = "change-me"
+  enabled       = true
```

Comment:
Suggestion: remove enabled from this example, which purpose should be to show the simplest possible way of creating a user.

Also, would be good to have comments above two examples explaining the purpose (and what's the idea behind the second one).